Update README.md: add link to osrd.fr (#3591)
OpenRailAssociation/osrd: README.md

It's free and open-source forever!

> More detailed documentation is available on [the OSRD website](https://www.osrd.fr).

Reviewer: ```suggestion
Learn more about the project on [osrd.fr](https://osrd.fr/en/).
```
